testers/specs.json

```diff
@@ -0,0 +1,6 @@
+{
+  "path_to_uam": "/Users/mschwa/Desktop/markus-autotesting/testers/python/uam-git",
+  "test_timeout": 10,
+  "global_timeout": 3600,
+  "feedback_file": null
+}
```

Comment:
What is this file?

Reply:
I assumed that was there from before. If it's not being used I'll remove it